### DISCUSSION TITLE
docker compose is not working

### DISCUSSION THREAD

**Discussion by leohao1018** [socketio/socket.io#4669] [Q&A] [answered]
[socket.io](https://github.com/socketio/socket.io)/[examples](https://github.com/socketio/socket.io/tree/main/examples)/cluster-nginx/
when i run  [docker-compose up -d]  is not working  
`
docker-compose up -d
`
`
(root) Additional property server-george is not allowed
`

**Comment by darrachequesne** [comment, marked as answer]
Hi! This should be fixed now, we have migrated the examples to the new syntax: https://github.com/socketio/socket.io/tree/main/examples/cluster-nginx

Thanks for the heads-up! :+1:
